### PR TITLE
Spanner plugin e2e test scenarios

### PR DESCRIPTION
**Pull request by sawantpritam**
@rmstar @itsankit-google Please review.

### CODE REVIEW

File: src/e2e-test/features/spanner/sink/GCSToSpanner.feature [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    Then Verify the preview of pipeline is "success"
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+
+  @GCS_CSV_TEST @SPANNER_SINK_NEWDB_TEST
+  Scenario: To verify data is getting transferred successfully from GCS to Spanner new database
```
Comment (rmstar) [inline src/e2e-test/features/spanner/sink/GCSToSpanner.feature:34]: Can we also add a test for the case where the database already exists, but table doesn't exist?

File: src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java [hunk 2]
```diff
@@ -191,4 +206,107 @@ private static void deleteGCSBucket(String bucketName) {
...
+          if (e.getMessage().contains("NOT FOUND")) {
+            BeforeActions.scenario.write("Spanner instance " + spannerInstance + " does not exist.");
+          }
+        }
+      }));
+      firstSpannerTestFlag = false;
+
+      //Create spanner instance
+      String spannerInstanceId = "e2e-" + (new SimpleDateFormat("yyyyMMdd-HH-mm-ssSSS").format(new Date()))
+        + (int) (Math.random() * (10000) + 1);
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java]: nit: don't need parantheses around 10000

File: src/e2e-test/features/spanner/sink/GCSToSpanner.feature (hunk 1 above)
Comment (sawantpritam) [reply src/e2e-test/features/spanner/sink/GCSToSpanner.feature:34]: `Scenario: To verify data is getting transferred successfully from GCS to Spanner existing database new table`

File: src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java (hunk 2 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java]: Done

**Comment by sawantpritam** [comment]
Spanner e2e test scenarios added in https://github.com/data-integrations/google-cloud/pull/906

**Pull request closed without merging**